_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -368,17 +302,18 @@ This [HTML document][] has a [keyboard shortcut][] using only a [printable chara
 </html>
 ```
 
-#### Failed Example 5
+#### Passed Example 8
```

Comment:
Order expectations (and their items) the same way your order examples.
Currently, expectations are ordered stop/focus/modifier/location and the examples stop/modifier/location/focus.